client/client-multi/client-c/src/kaa/platform-impl/esp8266/esp8266_key_utils.c

```diff
@@ -0,0 +1,12 @@
+#include <stdbool.h>
+#include <stddef.h>
+#include <platform/ext_key_utils.h>
+#include "rsa.h"
+
+
+void ext_get_endpoint_public_key(char **buffer, size_t *buffer_size, 
+                                            bool *needs_deallocation) {
```

Comment:
According to Kaa code style, opening brace should be on the new line.
Also the max line length is 120, so all parameters may be listed on one line.